challenge.rb

```diff
@@ -67,6 +92,12 @@ def ac_monitor current_temp, ac_working, desired_temp
 # and print out the result of `fizz_buzz_calculator` for that number.
 # This question will be graded independently of question four.
 
-# Replace this comment with your code.
+def fizz_buzz (min, max)
+ while min < max do
+    min += 1
+    return fizz_buzz_calculator(min)
```

Comment:
_Now_ you use return? (jk)
Make this one a `p`, `print`, or `puts`, and you've got the right answer.